### DISCUSSION TITLE
API support

### DISCUSSION THREAD

**Discussion by nemec** [TandoorRecipes/recipes#818] [Q&A] [answered]
Hi, is there an API to give programmatic access to data like recipes and meal planning? I see there is no Home Assistant integration at the moment, but if there's an API I'd like to try and rig up some cards like "Tonight's Meal" with the existing tools.

**Comment by vabene1111** [comment, marked as answer]
almost every object used by tandoor is accessible via a standardized RESTful API. You can use the integrated API browser and API Documentation to explore the endpoints and test your requests. 

The only things not available via API are internal components and some logging stuff.

**Reply by nemec** [reply]
Perfect, thanks for the quick reply 😊
